FakeCode-Chicago.md

```diff
@@ -0,0 +1,5 @@
+# Chicago
+
+### Things to do
+-- still am thinking
```

Comment:
@FakeCode Lollapalooza is pretty good if you can stand the crowds :musical_keyboard: :guitar: :musical_note: